prac_07/myguitars.py

```diff
@@ -0,0 +1,52 @@
+"""
+prac_07 - myguitars.py
+"""
+
+import csv
+
+from prac_07.guitar import Guitar
+
+FILENAME = "guitars.csv"
+
+
+def main():
+    """Read guitars from a file and store them in a list of Guitar objects."""
+    parts = load_data_from_file(FILENAME)
+    guitars = [Guitar(name, year, cost) for name, year, cost in parts]
+    guitars.sort()
+    for guitar in guitars:
+        print(guitar)
+
+    # From prac_06
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar_to_add = Guitar(name, year, cost)
+        guitars.append(guitar_to_add)
+        print(f"{guitar_to_add} added.")
+        name = input("Name: ")
+
+    save_guitars_to_file(FILENAME, guitars)
+
+
+def load_data_from_file(filename):
```

Comment:
Could we instantiate the Guitar objects in this function?

Reply:
Yes we can, that is a great idea!